godot-macros/src/method_registration/mod.rs

```diff
@@ -0,0 +1,137 @@
+/*
+* This Source Code Form is subject to the terms of the Mozilla Public
+* License, v. 2.0. If a copy of the MPL was not distributed with this
+* file, You can obtain one at https://mozilla.org/MPL/2.0/.
+*/
+pub mod register_method;
+pub mod virtual_method_callback;
```

Comment:
I would rather re-export those, and keep their modules private. Otherwise, this granularity just makes usage more complicated.